src/api.js

```diff
@@ -2,18 +2,23 @@
 
 const path = require("path")
 const fs = require("fs")
+const _ = require("lodash")
 
 const bundling = require("./bundling")
 const karma = require("./karma")
 const devServer = require("./dev_server")
+const sdk = require("./sdk")
 
 const API = {}
+const sdkClone = _.cloneDeep(sdk)
+delete sdkClone.test
```

Comment:
add a comment that we are sanitizing the public API.
and preventing th exposure of "internal" methods